### PR TITLE
Copy edit introductory text.

### PR DESCRIPTION
**Pull request by nwalfield**

### CODE REVIEW

File: doc/level1.rst [hunk 1]
```diff
@@ -2,25 +2,32 @@ Autocrypt Level 1: Enabling encryption, avoiding annoyances
...
-as though it were only capable of controlling a single e-mail account
-(see :ref:`multiaccounts` for more detail).
+specifies the requirements for Autocrypt Level 1 capable mail apps.
+
+The design of Autocrypt Level 1 is driven by usability concerns, and
+the realities of incremental deployment.  First, an Autocrypt-capable
+client can coexist with transitional mail clients.  That is, if
+someone uses an Autocrypt-capable mail client on her Desktop, she can
+continue to use a traditional mail client on her phone; she doesn't
+need to also use an Autocrypt capable client on phone.  (In fact, only
```
Comment (pbrunschwig) [inline doc/level1.rst]: ... need to also use an Autocrypt capable client on **her** phone.

File: doc/level1.rst [hunk 2]
```diff
@@ -2,25 +2,32 @@ Autocrypt Level 1: Enabling encryption, avoiding annoyances
...
+a single Level 1 capable mail client can be used per mail account.)
```
Comment (pbrunschwig) [inline doc/level1.rst]: Not necessarily. The Setup Message allows to have more then 1 Autocrypt MUA per mail account

File: doc/level1.rst [hunk 3]
```diff
@@ -2,25 +2,32 @@ Autocrypt Level 1: Enabling encryption, avoiding annoyances
...
-experiences as soon as possible.
-
-Throughout this document, we refer to a mail app or Mail User Agent (MUA)
-as though it were only capable of controlling a single e-mail account
-(see :ref:`multiaccounts` for more detail).
+specifies the requirements for Autocrypt Level 1 capable mail apps.
+
+The design of Autocrypt Level 1 is driven by usability concerns, and
+the realities of incremental deployment.  First, an Autocrypt-capable
+client can coexist with transitional mail clients.  That is, if
```
Comment (nwalfield) [inline doc/level1.rst]: s/transitional/traditional/

**Comment by nwalfield** [comment]
Although this tries to be an isomorphism in terms of content and just cleanup the language, it has proven highly controversial, as such, I retract this patch.

**Pull request closed without merging**